homeassistant/components/zha/config_flow.py

```diff
@@ -0,0 +1,55 @@
+"""Config flow for ZHA."""
+from collections import OrderedDict
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import (
+    DOMAIN, BAUD_RATES, CONF_BAUDRATE, CONF_DATABASE, CONF_RADIO_TYPE,
+    CONF_USB_PATH, RadioType
+)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class ZhaFlowHandler(config_entries.ConfigFlow):
+    """Handle a config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_user(self, user_input=None):
+        """Handle a zha config flow start."""
+        if self._async_current_entries():
+            return self.async_abort(reason='single_instance_allowed')
+
+        errors = {}
+
+        if user_input is not None:
+            return self.async_create_entry(
+                title=user_input[CONF_USB_PATH], data=user_input)
+
+        fields = OrderedDict()
+        fields[vol.Required(CONF_DATABASE)] = str
```

Comment:
There should be no reason to ask for a database path, we should just hardcode it? Only allow tweaking it via `configuration.yaml`

Reply:
Can we provide a default value instead of hardcoding it? Lets say as a "power" user I prefer when the choice is left to me, but I completely understand the effort to make it more user friendly.

Reply:
No default value. This should not be asked of the user. Allow overriding it via `configuration.yaml`